packages/cli/src/import/importAssetsFromCsv.ts

```diff
@@ -0,0 +1,129 @@
+import globby from 'globby';
+import upath from 'upath';
+import createSpinner from 'ora';
+import { AxiosInstance } from 'axios';
+import path from 'path';
+import pMap from 'p-map';
+import { retry } from '@lifeomic/attempt';
+
+import * as log from '../log';
+import { ImportAssetsParams } from './importAssets';
+import { readFileFromPath, getCsvAssetsDirectory } from '../fileSystem';
+import { createApiClientWithApiKey, getApiBaseUrl } from '@jupiterone/integration-sdk-runtime';
+import { sanitizeContent } from '../export/util';
+import { pause } from '../pause';
+
+export type ImportAssetsFromCsvParams = ImportAssetsParams & { apiKey: string };
+
+const J1_ENDPOINT = getApiBaseUrl({ dev: !!process.env.JUPITERONE_DEV });
+
+async function waitForSyncCompletion({ jobId, apiClient, progress }) {
+  let status;
+  do {
+    const { data: { job: jobStatus } } = await apiClient.get(`/persister/synchronization/jobs/${jobId}`);
+
+    progress(jobStatus);
+    status = jobStatus.status;
+    await pause(5000);
+  } while (status !== 'FINISHED')
+}
```

Comment:
Nice. At some point it would be nice to have this in another module so that it can be shared by the integration-sdk and this cli.